opm/utility/ECLPvtCurveCollection.hpp

```diff
@@ -91,6 +91,45 @@ namespace Opm { namespace ECLPVT {
                     const ECLPhaseIndex phase,
                     const int           activeCell) const;
 
+        /// Compute a single dynamic property in a single active cell for a
+        /// collection of cell states.
+        ///
+        /// \param[in] property Named property.  Must be one of \code
+        ///    RawCurve::FVF \endcode or \code RawCurve::Viscosity \endcode.
+        ///    All other values return an empty result.
+        ///
+        /// \param[in] phase Phase for which to compute the property value.
+        ///    Must be \code ECLPhaseIndex::Vapour \endcode or \code
+        ///    ECLPhaseIndex::Liquid \endcode.  All other values return an
+        ///    empty result.
+        ///
+        /// \param[in] activeCell Index of particular active cell in model.
+        ///
+        /// \param[in] phasePress Sequence of phase pressure values
+        ///    pertaining to \p activeCell.  Could, for instance, be the
+        ///    entire time-series of oil pressure values in that cell.
+        ///
+        /// \param[in] mixRatio Sequence of phase mixing ratio values
+        ///    pertaining to \p activeCell.  Could, for instance, be the
+        ///    entire time-series of dissolved gas/oil ratio values in that
+        ///    cell.  Must be empty or match the size of \p phasePress.  If
+        ///    empty, treated as \code
+        ///    std::vector<double>(phasePress.size(), 0.0) \endcode which is
+        ///    typically appropriate only for dry gas or dead oil cases.
+        ///
+        /// \return Sequence of dynamic property values corresponding to the
+        ///    requested property name of the identified phase in the
+        ///    particular active cell.  Empty for invalid requests, number
+        ///    of elements equal to \code phasePress.size() \endcode
+        ///    otherwise.
```

Comment:
Note that output is *not* given in SI, but in the same unit used in the deck (similar to how the PVT property curves are output). At the same time, the input *must* be in SI (phasePress and mixRatio, which is rs or rv).